lib/cc/services/github_pull_requests.rb

```diff
@@ -50,10 +54,31 @@ def receive_pull_request
     response
   end
 
+  def receive_pull_request_coverage
+    setup_http
+    state = @payload["state"]
+
+    if state == "success"
```

Comment:
Good catch. Might want to move the existing `if update_status?` check up a level so we can continue to reuse that `#update_status` method.

Check `if update_coverage_status` in `#receive_pull_request_coverage` and `if update_status` in `#receive_pull_request`.